nixos/modules/services/mail/listmonk.nix

```
@@ -54,7 +54,7 @@ let
 
       smtp = mkOption {
         type = listOf (submodule {
-          freeformType = with types; attrsOf (oneOf [ str int bool ]);
+          freeformType = with types; attrs;
```

```suggestion
          freeformType = with types; attrsOf anything;
```
Maybe this allows more flexibility while still retaining some types.

---

done